corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -87,7 +87,7 @@ def test_duplicate_claim(self):
     @run_with_all_backends
     def test_search_endpoint(self):
         known_result = (
-            '<fixture id="case">'  # (this is not the case type)
+            '<results id="case">'  # ("case" is not the case type)
```

Comment:
Maybe not worth changing, but on the mobile side we don't use `id="case"` for anything